File: Practicas/RobocodeLabo-1.10.0/RobocodeLabo/src/laboratorio/estrategas/EstrategaChill.java
package estrategas;

import estrategias.Estrategia;
import estrategias.EstrategiaAntiWalls;
import estrategias.EstrategiaWalls;
import laboratorio.NicoustinRobot;

public class EstrategaChill implements Estratega {
    public Estrategia chooseStrategy(NicoustinRobot robot) {
        if (robot.others > 3){
            return new EstrategiaWalls();
        } else {
            return new EstrategiaAntiWalls();
        }
    }
}

File: Practicas/Robocode-Kotlin/src/estrategas/EstrategaChill.kt
package estrategas

import laboratorio.NicoustinRobot
import estrategias.Estrategia
import estrategias.EstrategiaWalls
import estrategias.EstrategiaAntiWalls

class EstrategaChill : Estratega {
    
    override fun chooseStrategy(robot: NicoustinRobot): Estrategia {
        val enemigos = robot.others
        
        return if (enemigos > 3) {
            EstrategiaWalls()
        } else {
            EstrategiaAntiWalls()
        }
    }
}

